### PR TITLE
Guzzle Implementation

### PR DESCRIPTION
**Pull request by tomkel5**
This pull request introduces Guzzle as a means of facilitating API calls to the BSD Tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,33 @@
 Blue State Digital PHP API Client
 =================================
 
-Installation
-----------
+This library provides an interface to the BSD Tools.
+
+Example usage:
+
+```
```
Comment (chuck) [inline README.md]: Is there a "use'" call missing here?

**Comment by chuck** [comment]
To be clear I haven't read all of the code closely, but if it works, seems fine. Since this will be public I'd ask that we do an extra pass for making brace style consistent and cleaning up some extra newlines, though.

File: README.md (hunk 1 above)
Comment (tomkel5) [inline README.md]: Fixed c943d01

**Comment by tomkel5** [comment]
@chuck I went through and cleaned up a little 236f67b, then I made the braces all PSR-2 compliant on cd11200.

I've also verified that the `ExampleTest` produces the correct output when provided with a valid API User ID, secret, and base URL.

**Pull request merged**